### ISSUE TITLE
An error occurs when using the ancient seed

### ISSUE THREAD

**Issue by Jeesson** [probablykory/useful-trophies#1]
**Game version** `0.217.14 Hildir's Update`
**Mod version** `1.0.0`
**r2modman profile code** `018a6a2a-793e-a609-ab6d-8a7253667483`

```
[Error  : Unity Log] MethodAccessException: Method `Player.GetPlayersInRange(UnityEngine.Vector3,single,System.Collections.Generic.List`1<Player>)' is inaccessible from method `UsefulTrophies.Patches.ApplyStatusEffect(StatusEffect,single,UnityEngine.Vector3)'
Stack trace:
UsefulTrophies.Patches.ApplyStatusEffect (StatusEffect statusEffect, System.Single time, UnityEngine.Vector3 position) (at <d7728bf4f9dc419cbb7991eb3fcfdce8>:0)
UsefulTrophies.Patches.HumanoidUseItem (Humanoid __instance, Inventory inventory, ItemDrop+ItemData item, System.Boolean fromInventoryGui, Inventory ___m_inventory, ZSyncAnimation ___m_zanim) (at <d7728bf4f9dc419cbb7991eb3fcfdce8>:0)
(wrapper dynamic-method) Humanoid.DMD<Humanoid::UseItem>(Humanoid,Inventory,ItemDrop/ItemData,bool)
(wrapper dynamic-method) InventoryGui.DMD<InventoryGui::OnRightClickItem>(InventoryGui,InventoryGrid,ItemDrop/ItemData,Vector2i)
InventoryGrid.OnRightClick (UIInputHandler element) (at <447b19510f604ca3807cd6cce3a7c1ef>:0)
UIInputHandler.OnPointerDown (UnityEngine.EventSystems.PointerEventData eventData) (at <1cc5e200d8fe4d129ecc905a7d16b102>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute (UnityEngine.EventSystems.IPointerDownHandler handler, UnityEngine.EventSystems.BaseEventData eventData) (at <e97aac8d17514bea83fadfd039e9187a>:0)
UnityEngine.EventSystems.ExecuteEvents.Execute[T] (UnityEngine.GameObject target, UnityEngine.EventSystems.BaseEventData eventData, UnityEngine.EventSystems.ExecuteEvents+EventFunction`1[T1] functor) (at <e97aac8d17514bea83fadfd039e9187a>:0)
UnityEngine.EventSystems.EventSystem:Update()

```

**Comment by probablykory** [comment]
Thanks for the report.  I've confirmed this is another issue of the original code.  I'll publish a fix later today.

**Comment by probablykory** [comment]
Now resolved in 1.0.1
